website/docs/about/contribution.md

```diff
@@ -25,6 +25,21 @@ Possible forms of collaboration include, but are not limited to:
 
 We support a small number of autonomous but coordinated direct contributions. These include bug fixes in the code, new icons, documentation improvements, and small enhancements to existing components.
 
+### Contribute a new icon in Figma
+
+If you’d like to add a new icon to the [icon library](/icons/library), follow the steps in the [Figma library README](https://www.figma.com/file/MYiw4kiVpunIMMw0sBkE1t/%E2%9C%8F%EF%B8%8F-Flight-Development?node-id=566%3A1129&t=Bbflj3UUaWVyhamn-4).
+
+### Contribute in code
+
+To work in one of the Helios code packages, consult the corresponding `CONTRIBUTING.md` file for:
+- [Tokens](https://github.com/hashicorp/design-system/blob/main/packages/tokens/CONTRIBUTING.md)
+- [Ember Flight Icons](https://github.com/hashicorp/design-system/blob/main/packages/ember-flight-icons/CONTRIBUTING.md)
+- [Components](https://github.com/hashicorp/design-system/blob/main/packages/components/CONTRIBUTING.md)
```

Comment:
In case we want the text to be a bit more compact, we can replace the list with just inline text/links:
```suggestion
To work in one of the Helios code packages, consult the corresponding `CONTRIBUTING.md` file for [Tokens](https://github.com/hashicorp/design-system/blob/main/packages/tokens/CONTRIBUTING.md), [Ember Flight Icons](https://github.com/hashicorp/design-system/blob/main/packages/ember-flight-icons/CONTRIBUTING.md), and [Components](https://github.com/hashicorp/design-system/blob/main/packages/components/CONTRIBUTING.md)
```